desktop/core/src/desktop/js/ext/aceHelper.ts

```diff
@@ -71,7 +71,7 @@ const DIALECT_ACE_MODE_MAPPING: { [dialect: string]: string } = {
   'phoenix': 'ace/mode/phoenix',
   'presto': 'ace/mode/presto',
   'solr': 'ace/mode/solr',
-  'sql': 'ace/mode/sql'
```

Comment:
I think it'll use 'sql' as the default in some cases, better keep it to be safe.